app/va/va_profile/va_profile_client.py

```
@@ -165,35 +170,38 @@ def get_email(self, va_profile_id: RecipientIdentifier) -> str:
         self.statsd_client.incr('clients.va-profile.get-email.failure')
         self._raise_no_contact_info_exception(self.EMAIL_BIO_TYPE, va_profile_id, contact_info.get(self.TX_AUDIT_ID))
 
-    def has_valid_mobile_telephone_classification(self, telephone: Telephone) -> bool:
+    def has_valid_mobile_telephone_classification(self, telephone: Telephone, contact_info: ContactInformation) -> bool:
         """
         Args:
           telephone (Telephone): telephone entry from ContactInformation object retrieved from Vet360 API endpoint
 
         Returns:
           bool - if AWS-classified telephone is a valid sms recipient (if nonexistent, return True)
+
+        Raises:
+          InvalidPhoneNumberException - if AWS-classified telephone is not a valid sms recipient
         """
         classification = telephone.get('classification', {})
         classification_code = classification.get('classificationCode', None)
-        if classification_code is not None:
-            if classification_code not in VALID_PHONE_TYPES_FOR_SMS_DELIVERY:
-                self.logger.debug(
-                    'V3 Profile -- Phone classification code of %s is not a valid SMS recipient (VA Profile ID: %s)',
-                    classification_code,
-                    telephone['vaProfileId'],
-                )
-                return False
+        if classification_code is None:
+            # fall back, if no phone number classification is present
+            self.logger.debug(
+                'V3 Profile -- No telephone classification present, assuming the number is a valid SMS recipient (VA Profile ID: %s)',
+                telephone['vaProfileId'],
+            )
+            return True
```

Change from last time since I was in this function here: fail fast.